openfn/docker-compose.yml

```
@@ -0,0 +1,64 @@
+version: '3.9'
+
+services:
+  openfn:
+    image: ${OpenFn_IMAGE}
+    # This command runs a shell script that performs the following actions:
+    # 1. Executes the Lightning.Release.migrate() function to handle database migrations.
+    # 2. Sets up a user with the provided first name, last name, email, password, and role using the Lightning.Setup.setup_user function.
+    #    - The user details are hardcoded with the first name "Test", last name "User", email "root@openhim.org", and password "instant101".
+    #    - The role assigned to the user is "superuser".
+    #    - The function also takes an API key from the environment variable ${OPENFN_API_KEY}.
+    #    - Additionally, it sets up a schema with the name "openhim ndr" and type "http", including credentials and a base URL from environment variables.
+    # 3. Starts the Lightning application using the /app/bin/lightning start command.
+    command: >
+      sh -c "/app/bin/lightning eval 'Lightning.Release.migrate()' && /app/bin/lightning eval 'Lightning.Setup.setup_user(%{first_name: \"Test\",last_name: \"User\",email: \"root@openhim.org\",password: \"instant101\", role: :superuser}, \"${OPENFN_API_KEY}\" ,[%{name: \"openhim ndr\", schema: \"http\", body: %{username: \"${FHIR_SERVER_USERNAME}\", password: \"${FHIR_SERVER_PASSWORD}\", baseUrl: \"${FHIR_SERVER_BASE_URL}\"}}])' && /app/bin/lightning start"
+    deploy:
+      resources:
+        limits:
+          cpus: '${DOCKER_WEB_CPUS:-0}'
+          memory: '${DOCKER_WEB_MEMORY:-0}'
+    environment:
+      - DATABASE_URL=${DATABASE_URL}
+      - DISABLE_DB_SSL=${DISABLE_DB_SSL}
+      - IS_RESETTABLE_DEMO=${IS_RESETTABLE_DEMO}
+      - LISTEN_ADDRESS=${LISTEN_ADDRESS}
+      - LOG_LEVEL=${LOG_LEVEL}
+      - ORIGINS=${ORIGINS}
+      - PRIMARY_ENCRYPTION_KEY=${PRIMARY_ENCRYPTION_KEY}
+      - SECRET_KEY_BASE=${SECRET_KEY_BASE}
+      - WORKER_RUNS_PRIVATE_KEY=${WORKER_RUNS_PRIVATE_KEY}
```

```suggestion
      - WORKER_RUNS_PRIVATE_KEY=${OPENFN_WORKER_RUNS_PRIVATE_KEY}
```
Could we pre-append **OPENFN** to environment variables that are specifically for OpenFN? This will help manage the environments easily from a single environment file.